tests/bin/initialize.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 npm run env run tests-wordpress "chmod -c ugo+w /var/www/html"
 npm run env run tests-cli "wp rewrite structure '/%postname%/' --hard"
-npm run env run tests-cli "wp post create --post_type=page --post_title='Page with special characters' --post_status='publish'"
\ No newline at end of file
+npm run env run tests-cli "wp post create --post_type=page --post_title='Page with special characters' --post_status='publish'"
+npm run env run tests-cli "wp post create --post_type=page --post_title='Page with a format and special characters' --post_status='publish'"
```

Comment:
Thanks for the changes! It is working for me now 👍 

Can you also elaborate why was this command added?

Reply:
@Sidsector9 appologies, i was trying to write a new cypress test for the change however, cypress was not playing nice, i thought i cleaned this up!

Reply:
fixed!